src/modules/auth/auth.controller.ts

```diff
@@ -31,4 +42,21 @@ export class AuthController {
   login(@Body() loginDto: LoginDto) {
     return this.authService.login(loginDto);
   }
+
+  @Get('google/redirect')
+  @UseGuards(AuthGuard('google'))
+  @ApiOperation({
+    summary: 'Google Authentication Callback',
+    description: 'Callback for Google OAuth2 login',
+  })
+  @ApiResponse({
+    status: 200,
+    description: 'User info from Google',
+  })
+  async googleAuthRedirect(@GetUser() user: GoogleUser) {
+    if (!user) {
```

Comment:
Redundant check since in the decorator we are already throwing an error

Reply:
fixed